pkg/model/estimator/local/ratio_process.go

```diff
@@ -27,8 +27,51 @@ import (
 	"k8s.io/klog/v2"
 
 	collector_metric "github.com/sustainable-computing-io/kepler/pkg/collector/metric"
+	"github.com/sustainable-computing-io/kepler/pkg/config"
 )
 
+// getTransferredDataFromMetric is used when there is Hardware counter so we have to use cgroup or eBPF (CPUTime)
+// directly, currently for CoreUsageMetric we use cputime and for DRAM, use cgroup dram info
+func getTransferredDataFromMetric(containerMetrics *collector_metric.ContainerMetrics, usageMetrics string) float64 {
```

Comment:
Why the name `TransferredData`?

Reply:
I can make it another name, like getCgroupDataFromMetrics

Reply:
what about: `getDataFromAlternativeMetric()`